modules/ROOT/pages/a11ychecker.adoc

```diff
@@ -393,3 +383,20 @@ console.log(issues);
   }
 ]
 ----
+
+[[issue]]
+=== The `+issue+` object
+
+Each `rule` in a11ychecker will generate an issue on any element which matches it. This issue is available as an object in a number of APIs, and contains the specific data used in the a11ychecker dialog, pertaining to the element which triggered it. With `a11ychecker_filter_issue`, these issues can be parsed before they are classified as issues, with the data provided in the `issue` object used in the callback function. 
+
+The `+issue+` object is provided to any function implementation of this option. This object provides relevant data pertaining to the triggered issue:
+
+* `+id+` : `+String+` A11ychecker issue identifier used by {productname}, such as D1, T4A. For a description and other details about the issue, see xref:accessibility-rules[Accessibility Rules].
+
+* `+description+` : `+String+` description of the issue, as will be displayed in the dialog.
+
+* `+severity+` : `+String+` severity level of the issue, either `+info+`, `+warning+`, or `+error+`.
```

Comment:
```suggestion
* `+description+` : `+String+` description of the issue; as will be displayed in the dialog.

* `+severity+` : `+String+` severity level of the issue; either `+info+`, `+warning+`, or `+error+`.
```

A semi-colon to delimit the two descriptive clauses is a stronger demarcator and encourages the reader to take a mental breath between each clause.

Reply:
Good idea, thanks